workflows/analyze.smk

```diff
@@ -18,6 +31,8 @@ rule analyze_metric:
         mcmc_config_id = "MC.*",
         # metric wildcard cannot be log_prob
         metric=r"(?!(log_prob))\w+",
+    resources:
+        mem_mb=get_mem_mb,
```

Comment:
Dynamic Snakemake Ram - Part 2: Here I assign the dynamic ram. - It works and allows me to set ram rather low as a default.